themes/default/content/what-is/run-aws-s3-ls-with-dynamic-credentials.md

```diff
@@ -83,15 +83,19 @@ values:
 
 ### Step 5: Run the aws s3 ls command
 
-First make sure that your local environment does not have any AWS credentials configured. You can do this by running the `aws s3 ls` command normally as shown below:
+First check that your local environment does not have any AWS credentials configured by running the `aws configure list` command as shown below:
```

Comment:
The instructions are written this way because if they do have AWS credentials configured and they run aws commands it's going to work regardless. This is supposed to help them validate that it is running correctly because of esc run and not because of any locally configured AWS credentials. Changing to "if you dont have aws credentials" won't capture that.